client/Meeting.coffee

```diff
@@ -154,6 +155,17 @@ export Meeting = ->
       if location.hash
         history.replace "/m/#{meetingId}"
       Session.set 'currentRoom', undefined
+    
+    if model.getRoot().getChildren().every (tabset) -> tabset.getChildren().length == 0
+      welcomeTabJson =
+        id: 'welcome'
+        type: 'tab'
+        name: 'Welcome'
+        component: 'Welcome'
+        enableClose: true
+        enableDrag: true
```

Comment:
Can't this be added directly to `initModel`? Or is the point to re-add it when someone closes it and everything else? (Not sure whether that's annoying...)  I had in mind that `enableClose` would be `false`...

You also don't need to set these `enable`s explicitly to `true`; they're the default.

Reply:
I was thinking an unclosable tab would be even more annoying

Reply:
As discussed: then probably shouldn't magically come back when you leave all rooms. Can always reload Comingle if you really want the welcome tab back.

Reply:
Oops, reloading won't restore it, because of localstorage magic. Maybe it's better to leave this as is...

Reply:
Oops again. We don't store the layout for meetings, only tabs within meetings, so this will actually work.